src/Services/CampaignService.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Mailjet\LaravelMailjet\Services;
+
+use \Mailjet\Resources;
+use \Mailjet\Response;
+use Mailjet\LaravelMailjet\Services\MailjetService;
+use Mailjet\LaravelMailjet\Model\Campaign;
+use Mailjet\LaravelMailjet\Contracts\CampaignContract;
+use Mailjet\LaravelMailjet\Exception\MailjetException;
+
+/**
+ * https://dev.mailjet.com/email-api/v3/campaign/
+ * list/view/update
+ */
+class CampaignService implements CampaignContract {
```

Comment:
Could be useful to have `findByNewsletterId` since this is a common request? https://dev.mailjet.com/guides/#retrieve-campaign-information-for-a-particular-newsletter